apis/projectcontour/v1/httpproxy.go

```diff
@@ -1036,6 +1036,78 @@ type Service struct {
 	// Slow start will gradually increase amount of traffic to a newly added endpoint.
 	// +optional
 	SlowStartPolicy *SlowStartPolicy `json:"slowStartPolicy,omitempty"`
+	// The policy for managing outlier detection on a service.
+	// If not specified, the global OutlierDetection policy will be used.
+	// +optional
+	OutlierDetection *OutlierDetection `json:"outlierDetection,omitempty"`
+}
+
+// OutlierDetection defines the configuration for outlier detection on a service.
+type OutlierDetection struct {
```

Comment:
a link to the Envoy docs would be nice here

Reply:
Yes, I will add a link to envoy's documentation above the comment